### PR TITLE
Switched to rubric built in title and graded by

### PR DESCRIPTION
**Pull request by MitchelPaulin**
https://rally1.rallydev.com/#/57732444928ud/custom/373260458992?detail=%2Fuserstory%2F630669079213

Were making some changes in rubrics and switching to handling the graded by info on our end, this change is being controlled by the `rubrics-archive` feature flag. Since we are now handling the graded by info we can also handle displaying the title ourselves using the `show-title` tag.

### CODE REVIEW

**Review by imcdonald-d2l — APPROVED** [review]

**Review by devpow112 — APPROVED** [review]

File: components/right-panel/consistent-evaluation-rubric.js [hunk 1]
```diff
@@ -285,15 +285,18 @@ class ConsistentEvaluationRubric extends LocalizeConsistentEvaluation(RtlMixin(L
 		return `(${this.localize('notGradedSummary')})`;
 	}
 	_getRubrics() {
+
+		const rubricsArchivedFeature = window.D2L?.LP?.Web?.UI?.Flags?.Flag('rubrics-archive', false) ?? false;
```
Comment (amauryperez) [inline components/right-panel/consistent-evaluation-rubric.js:289]: Nice, I didn't know there was a way to use feature flags within web components. I think we used send FF values as attribute of the component.

**Review by amauryperez — APPROVED** [review]

File: components/right-panel/consistent-evaluation-rubric.js (hunk 1 above)
Comment (devpow112) [reply components/right-panel/consistent-evaluation-rubric.js:289]: It's a bit clunky but it works. This also works within FRA's (I added some sync functionality a while back)

Comment (amauryperez) [reply components/right-panel/consistent-evaluation-rubric.js:289]: Does it work in all supported browsers?

Comment (devpow112) [reply components/right-panel/consistent-evaluation-rubric.js:289]: Should be, it's used in a few spots already and hasn't thrown up any red flags.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility

**Pull request merged**